pom.xml

```diff
@@ -17,9 +17,9 @@
     <url>https://wiki.jenkins.io/display/JENKINS/Localization+Support+Plugin</url>
 
     <properties>
-        <revision>1.2</revision>
+        <revision>1.2.1</revision>
         <changelist>-SNAPSHOT</changelist>
-        <jenkins.version>2.173</jenkins.version>
+        <jenkins.version>2.249.3</jenkins.version>
```

Comment:
I wouldn't generally require a Jenkins version just a week old without a particular reason. [My recommendations](https://www.jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline/) currently are slightly more conservative. Why did you choose this version?

Reply:
I chose it because it was the newest version at the time. After reading the doc you provided, I now realize that was a mistake. Thanks for informing me.

>Do not use versions no longer supported by the update center, which is currently anything older than 2.199 for weekly releases, and 2.190.1 for LTS releases.

Could I replace `2.249.3` with `2.190.1` or do you want it to stay at 2.173?

Reply:
2.190.1 is a reasonable, conservative choice.